web/menu.jspf

```diff
@@ -202,10 +204,12 @@ org.opensolaris.opengrok.web.Util"
     </tbody>
 </table>
 <div id="form-controls">
+    <input tabindex="9" class="submit btn" type="button" value="Search All Projects" onclick="javascript: SearchAll();"/>
```

Comment:
Firstly this should be after the `Search` button to avoid confusion with already existing button.

Secondly I would use just `Search All` text or `Search all`